kolibri/plugins/coach/assets/src/views/quizzes/QuizSummaryPage/QuizOptionsDropdownMenu.vue

```diff
@@ -41,7 +41,7 @@
           },
           { label: this.coreString('deleteAction'), value: 'DELETE' },
         ];
-        if (!this.exam?.archive) {
+        if (!this.exam?.data_model_version > 2) {
```

Comment:
Looks like the `!` here is making for the condition being the opposite of what is expected as we only want the `editAction` to show when the quiz is `data_model_version > 2`. 

I think this is the cause of @radinamatic 's last report